### ISSUE TITLE
Delete translation jobs?

### ISSUE THREAD

**Issue by gforcada** [plone/jenkins.plone.org#351]
We have a few jobs in Jenkins that look for missing translations and such:

https://jenkins.plone.org/view/Translations/

Is anyone paying attention to them, or they can be removed? 🤔 

With `plone/meta` whenever a new PR is created, there is a check (with `pre-commit`) to check if there are strings not marked for translation... not sure if this is enough 🤷🏾

**Comment by erral** [comment]
Yes, we can remove the job that checks for missing i18n tags, but not the one that extracts the statistics.

Until Plone organization is fully validated in Weblate, I would like to keep the stats in Jenkins as they are now.

**Comment by gforcada** [comment]
So the ones that are broken we are meant to keep, and the ones that are running fine we no longer need them? 😆 

I guess is a small fix to get the broken ones fixed, well, at least the Plone backend ones, the volto one, no idea 😅

**Comment by davisagli** [comment]
If they need work anyway, maybe it makes more sense to move them to Github Actions instead of fixing the jenkins jobs?

**Comment by gforcada** [comment]
Sure, I don't have any strong desire to keep things in jenkins 😄 though, while nobody offers another solution (GHA or something else) we should keep that running if they are needed
